job-dsls/jobs/seed_job.groovy

```diff
@@ -61,7 +61,6 @@ job("a-seed-job") {
                     "job-dsls/jobs/**/dailyBuild_pipeline.groovy\n" +
                     "job-dsls/jobs/**/dailyBuild_prod_pipeline.groovy\n" +
                     "job-dsls/jobs/**/deploy_jobs.groovy\n" +
-                    "job-dsls/jobs/**/kie_docs_pr.groovy\n" +
```

Comment:
why is kie-docs PR job removed?

Reply:
because this is now created by the new_pr_jobs.groovy